ui/app/templates/components/console/log-help.hbs

```diff
@@ -25,6 +25,6 @@ Web REPL Commands:
 
 </pre>
   <p class="console-ui-panel-intro is-font-mono has-bottom-margin-s">
-    For more detailed documentation, see the <Hds::Link::Inline @href={{doc-link "/vault/docs/commands/web"}}>HashiCorp Developer site</Hds::Link::Inline>.
```

Comment:
Thanks for fixing! I was wondering why the redirect was working appropriately [here](https://github.com/hashicorp/vault/blob/main/ui/app/templates/components/console/ui-panel.hbs#L24) and noticed there that `command` is singular and here it's plural